src/Examples/Flowsheets/power_generation/sofc/sofc.ipynb

```diff
@@ -0,0 +1,2111 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8c5f00c8",
+   "metadata": {},
+   "source": [
+    "# SOFC Power Plant Example\n",
+    "\n",
+    "This notebook runs the SOFC power plant model over a range of net power outputs from 650 MW to 200 MW. The SOFC model is based on case ANGFC3B in the NETL report \"Techno-Economic Analysis of Natural Gas Fuel Cell Plant Configurations\" April 2022\n",
+    "https://www.netl.doe.gov/projects/files/TechnoEconomicAnalysisofNaturalGasFuelCellPlantConfigurations_043022.pdf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "db0e6427",
+   "metadata": {},
+   "source": [
+    "## Module Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "3dc42d5d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "In C:\\Users\\alxor\\Miniconda3\\lib\\site-packages\\matplotlib\\mpl-data\\stylelib\\_classic_test.mplstyle: \n",
```

Comment:
These deprecation warnings probably need to be addressed - we don;t want these showing up in examples if possible.